content/publication_resources/bibtex/lahti.bib

```diff
@@ -15,53 +15,61 @@ @Article{
   journal =	 {Nature Biotechnology},
   year =	 2023,
+  month =	 07,
   doi =		 {10.1038/s41587-023-01845-1},
   keywords =	 {bioscience}
 }    
 
 
 @Article{Gao2023,
-  author =	 {Yu Gao, Yağmur Şimşek, Emma Gheysen, Tuomas Borman, Yi Li, 
-                Leo Lahti, Karoline Faust, Daniel Rios Garza},
-  title =	 {miaSim: an R/Bioconductor package to easily simulate microbial community dynamics},
+  author =	 {Yu Gao and Yağmur Şimşek and Emma Gheysen and Tuomas Borman and
+                Yi Li and Leo Lahti and Karoline Faust and Daniel Garza},
+  title =	 {{miaSim: an R/Bioconductor package to easily simulate microbial community dynamics}},
   journal =	 {Methods in Ecology and Evolution},
   year =	 2023,
-  doi =		 {0.1111/2041-210X.14129},
+  month = 05,
+  doi =		 {10.1111/2041-210X.14129},
   keywords =	 {bioscience}
 }    
 
 
 @Article{Aatsinki2023,
-  author =	 {Anna-Katariina Aatsinki, Jetro J Tuulari, Eveliina J Munukka, 
-                Leo Lahti, Anniina Keskitalo, Henna-Maria Kailanto, Saara Nolvi,
-                    Noora M Scheinin, Jani Saunavaara, Riitta Parkkola, 
-                      John D Lewis, Niloofar Hashempour, Satu J Shulist, 
-                        Linnea Karlsson, Hasse Karlsson},
+  author =	 {Anna-Katariina Aatsinki and Jetro Tuulari and Eveliina Munukka 
+                and Leo Lahti and Anniina Keskitalo and Henna-Maria Kailanto and
+                  Saara Nolvi and Noora Scheinin and Jani Saunavaara and
+                    Riitta Parkkola and John Lewis and Niloofar Hashempour and 
+                      Satu Shulist and Linnea Karlsson and Hasse Karlsson},
   title =	 {Infant Left Amygdala Volume Is Negatively Associated with Fecal Microbiota Diversity},
   journal =	 {bioRxiv preprint},
   year =	 2023,
+  month = 04,
   doi =		 {10.1101/2023.04.23.537273},
   keywords =	 {bioscience}
 } 
 
-@Article{Aatsinki2023,
-  author =	 {Joonatan Palmu, Christin S Börschel, Alfredo Ortega-Alonso, 
-            Lajos Markó , Michael Inouye, Pekka Jousilahti , Rodolfo A Salido ,
-          Karenina Sanders, Caitriona Brennan , Gregory C Humphrey, Jon G Sanders,
-              Friederike Gutmann, Dominik Linz, Veikko Salomaa, Aki S Havulinna, 
-            Sofia K Forslund, Rob Knight, Leo Lahti, Teemu Niiranen, Renate B Schnabel},
+@Article{Palmu2023,
+  author =	 {Joonatan Palmu and Christin Börschel and Alfredo Ortega-Alonso 
+                and Lajos Markó and Michael Inouye and Pekka Jousilahti and 
+                  Rodolfo Salido and Karenina Sanders and Caitriona Brennan and
+                    Gregory Humphrey and Jon Sanders and Friederike Gutmann and
+                      Dominik Linz and Veikko Salomaa and Aki Havulinna and 
+                        Sofia Forslund and Rob Knight and Leo Lahti and 
+                          Teemu Niiranen and Renate Schnabel},
   title =	 {Gut microbiome and atrial fibrillation-results from a large population-based study},
   journal =	 {eBioMedicine},
```

Comment:
Journal name to double curly brackets because of the unconventional capitalizations